.github/workflows/gradlew-validate.yaml

```diff
@@ -0,0 +1,18 @@
+name: Gradle
```

Comment:
Not sure how GHA react to two workflows sharing one name? (+ IMHO a bit confusing :P )

Proposal rename:
* (this) Gradle Validate
* (gradle-update.yml) Gradle Update

Reply:
Ideally I wanted them in the same workflow file but that's impossible due too their different triggers. I chose to use the same name for both because GitHub always shows workflow and job name together.

Reply:
Okay, I see your use-case and if you say GH understands this then ignore my comment :wink:  :+1: